### PR TITLE
Add PyCon India 2016 keynote speakers announcement blog

### PR DESCRIPTION
**Pull request by meganeshkadam**

### CODE REVIEW

**Comment by chkumar246** [comment]
@kracekumar @vnbang2003 Can you merge this?

File: content/blog/announcing-keynote-speakers-for-pycon-india-2016.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Title: Announcing  keynote speakers for PyCon India 2016!
+Date: 2016-07-28 20:00
+Category: 2016
+Slug: announcing-keynote-speakers-for-pycon-2016
+Summary: Announcing PyCon India 2016 Keynote Speakers
+
+We're proud to share our best keynote lineup ever !
```
Comment (kracekumar) [inline content/blog/announcing-keynote-speakers-for-pycon-india-2016.md]: Please drop this claim (`best keynote lineup ever !`).

File: content/blog/announcing-keynote-speakers-for-pycon-india-2016.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**Announcing keynote-speakers for PyCon India 2016. See who will speak on our 2016 stage!**
```
Comment (kracekumar) [inline content/blog/announcing-keynote-speakers-for-pycon-india-2016.md]: Replace `keynote-speakers` with `keynoters`.

**Comment by kracekumar** [comment]
Please fix the comments and rest looks good.

**Pull request merged**